RELEASES.md

```diff
@@ -46,11 +46,13 @@ schedule, initially aiming for the bi-weekly releases.
 Release managers of stable releases are responsible for approving and merging backports, tagging
 stable releases and sending announcements about them. This role is rotating on a quarterly basis.
 
-| Quarter |       Release manager        |
-|:-------:|:----------------------------:|
-| 2020 Q1 | Piotr Sikora ([PiotrSikora]) |
-| 2020 Q2 | Piotr Sikora ([PiotrSikora]) |
-| 2020 Q3 | Yuchen Dai ([lambdai])       |
+| Quarter |       Release manager            |
+|:-------:|:--------------------------------:|
+| 2020 Q1 | Piotr Sikora ([PiotrSikora])     |
+| 2020 Q2 | Piotr Sikora ([PiotrSikora])     |
+| 2020 Q3 | Yuchen Dai ([lambdai])           |
+| 2020 Q4 | Christoph Pakulski ([cpakulski]) |
+| 2021 Q1 |                                  |
```

Comment:
Also, Rei Shimizu (@Shikugawa) agreed to be signed up in the next release. We can leave this updated at release 1.17 or just update now.

Reply:
Oh awesome, thanks @Shikugawa 
Also a huge thanks to @lambdai and @PiotrSikora for their work thusfar!  :-)